crypto/asn1/ameth_lib.c

```diff
@@ -264,6 +264,7 @@ void EVP_PKEY_asn1_copy(EVP_PKEY_ASN1_METHOD *dst,
 
     dst->pkey_size = src->pkey_size;
     dst->pkey_bits = src->pkey_bits;
+    dst->pkey_security_bits = src->pkey_security_bits;
 
     dst->param_decode = src->param_decode;
     dst->param_encode = src->param_encode;
```

Comment:
please add `dst->sig_print = src->sig_print;` after
the line copying `param_print` below

Reply:
It has been added.